app/scripts/directives/serviceIntegration.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('serviceIntegration', {
+    controller: [
+      'APIService',
+      'AuthorizationService',
+      'DataService',
+      'CatalogService',
+      ServiceIntegration
+    ],
+    controllerAs: '$ctrl',
+    bindings: {
+      integration: '<',
+      refApiObject: '<?',
+      serviceClasses: '<',
+      serviceInstances: '<',
+      isOverview: '<?'
+    },
+    templateUrl: 'views/directives/_service-integration.html'
+  });
+
+  function ServiceIntegration(APIService,
+                          AuthorizationService,
+                          DataService,
+                          CatalogService) {
+
+    var ctrl = this;
+    var serviceBindingsResource = {group: "servicecatalog.k8s.io", version: "v1beta1", resource: "servicebindings"};
```

Comment:
I think you can get this via ```APIService.getPreferredVersion('servicebindings')```

Reply:
cool, just testing this now, then will update the PR.